.github/workflows/test.yaml

```diff
@@ -0,0 +1,36 @@
+name: Test Cryptex Contracts
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    env:
+      ALCHEMY_KEY: _JDT5xiRpMR5r4HbVogiHG2sIdhhiksT
```

Comment:
@crisgarner Can you please add `ALCHEMY_KEY ` to the environment variable through settings? 
Also, I'm currently using my personal key. I'm going to delete this key once you add a new one the env vars.